routes/billingRoutes.js

```diff
@@ -5,14 +5,14 @@ const requireLogin = require('../middlewares/requireLogin');
 module.exports = app => {
   app.post('/api/stripe', requireLogin, async (req, res) => {
     const charge = await stripe.charges.create({
-      amount: 500, // 500 cents = 5 dollars
+      amount: 500,
```

Comment:
I think the comment is helpful! Are users only able to buy in increments of 5 at a time?

Reply:
yes... I think after Turing.. I want users to be able to be refunded or put their own amount.